README.md

```diff
@@ -28,13 +27,298 @@ pip install -U pyinstaller-hooks-contrib
 
 Either `a-package` works fine without a hook, or no-one has contributed hooks.
 If you'd like to add a hook, or view information about hooks,
-please see [the wiki](https://github.com/pyinstaller/pyinstaller-hooks-contrib/wiki).
+please see below.
 
 
 ## I want to help!
 
-Please start by providing pull requests and helping solve issues.
+If you've got a hook you want to share then great!
+The rest of this page will walk you through the process of contributing a hook.
+If you've been here before then you may want to skip to the [summary checklist](#summary)
+
+**Please provide one hook per pull request! If you have more than one then submit them in separate pull requests.**
+
+
+### Setup
+
+[Fork this repo](https://github.com/pyinstaller/pyinstaller-hooks-contrib/fork) if you haven't already done so.
+(If you have a fork already but its old, click the **Fetch upstream** button on your fork's homepage.)
+Clone and `cd` inside your fork by running the following (replacing `bob-the-barnacle` with your github username):
+
+```
+git clone https://github.com/bob-the-barnacle/pyinstaller-hoooks-contrib.git
+cd pyinstaller-hooks-contrib
+```
+
+Create a new branch for you changes (replacing `foo` with the name of the package):
+You can name this branch whatever you like.
+
+```
+git checkout -b hook-for-foo
+```
+
+If you wish to create a virtual environment then do it now before proceeding to the next step.
+
+Install this repo in editable mode.
+This will overwrite your current installation.
+(Not that you can reverse this with `pip install --force-reinstall pyinstaller-hooks-contrib`).
+
+```
+pip install -e .
+pip install -r requirements-test.txt
+pip install flake8
+```
+
+Note that on macOS and Linux, `pip` may by called `pip3`.
+If you normally use `pip3` and `python3` then use `pip3` here too.
+You may skip the 2<sup>nd</sup> line if you have no intention of providing tests (but please do provide tests!).
+
+
+### Add the hook
+
+Hooks live in the [src/_pyinstaller_hooks_contrib/hooks/stdhooks/](../master/src/_pyinstaller_hooks_contrib/hooks/stdhooks/) directory.
```

Comment:
Just wanted to confirm it was a purposeful decision to not talk about runtime hooks here

Reply:
I like to pretend that runtime hooks don't exist most of the time. 423b95f7b9f8ca626773e5e22424419a4fcb65ea adds a line on runtime hooks.